marathontvapp/expo-native-asset
# feat: support appiconset type

# Description

I've added `appiconset` asset type support to this plugin for A/B testing my Expo app. Additionally, I removed the ios and android folders, as these are dynamically created by expo prebuild.


## Related issues

No related issues


## How to test

```bash
cd example
npx expo prebuild -p ios --clean
```

Then, open Xcode and check `Images.xcassets`

---

I also tested this on my real Expo project.

![image](https://github.com/marathontvapp/expo-native-asset/assets/22038798/5daf64bb-bdce-4698-819a-1baf354eaf9c)
